templates/components/username.html.twig

```diff
@@ -3,6 +3,7 @@
         class="inline-flex items-center gap-1 {{ className|default('') }}"
         href="{{ path('forumify_forum_profile', { username: user.username }) }}"
         {{ stimulus_controller('forumify/forumify-platform/profile_preview', { userId: user.id }) }}
+        style="color: {{ user|role_color }};"
```

Comment:
If `user|role_color` is `null`, this will produce invalid HTML.

While it probably won't be visible because browsers are smart, we shouldn't be producing invalid styles by design.